pyvista/__init__.py

```diff
@@ -9,23 +9,272 @@
 # Load default theme.  Must be loaded first.
 from pyvista._version import __version__
 from pyvista.plotting import *
-from pyvista.utilities import *
-from pyvista.core import *
+from pyvista.utilities import (
+    # arrays,
+    cells,
```

Comment:
I added back `pyvista.cells` because `examples/00-load/linear-cells.py` references `pyvista.cells.Wedge`.

This must have caused broken intersphinx references, so I'll have to test locally if these emit warnings, or can be told to do so. I wouldn't want to end up with a bunch of silently broken doc links due to this PR.